src/DependencyInjection/Configuration.php

```diff
@@ -32,7 +32,7 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         /** @var ArrayNodeDefinition $rootNode */
         $rootNode = $treeBuilder->getRootNode();
-
+        // @phpstan-ignore-next-line
```

Comment:
@aryaantony92 We should add PhpStan Symfony Extension intead:
https://github.com/phpstan/phpstan-symfony

Reply:
@blankse Thanks! Done in [266aa3e](https://github.com/pimcore/personalization-bundle/pull/24/commits/266aa3e3e6e8160cae6f888e91801fcc390a0885)